apis/placement/v1beta1/appliedwork_types.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
A nit: would it be better if we have a comment here documenting the origin of this spec?

Reply:
we probably need to include the original copy right too